### ISSUE TITLE
Duplicate declaration: Package[gnupg] introduced with v2.0.0

### ISSUE THREAD

**Issue by saz** [voxpupuli/puppet-rvm#186]
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 6.27.0
- Ruby: 2.7
- Distribution: Ubuntu
- Module version: 2.0.0

## How to reproduce (e.g Puppet code you use)
Use `apt` and `rvm` at the same time

## What are you seeing
Error: Could not retrieve catalog from remote server: Error 500 on SERVER: Server Error: Evaluation Error: Error while evaluating a Resource Statement, Duplicate declaration: Package[gnupg] is already declared at (file: /etc/puppetlabs/code/environments/production/modules/apt/manifests/init.pp, line: 372); cannot redeclare (file: /etc/puppetlabs/code/environments/production/modules/gnupg/manifests/install.pp, line: 4) (file: /etc/puppetlabs/code/environments/production/modules/gnupg/manifests/install.pp, line: 4, column: 3) on node some-host.example.com

## What behaviour did you expect instead
Successful management of `rvm`

## Any additional information you'd like to impart
The `apt` module uses `ensure_packages(['gnupg'])` while the usage of `gnupg` module (introduced in 1c71b55f521cb03020a1e5bf9146b9184c22f6b8).

Except for either fixing the `gnupg` module (which is outdated anyways - last release 2016) or getting rid of it, I don't have any idea how to properly fix this.

**Comment by saz** [comment]
Maybe dropping `include gnupg` (it's only installing the package) and adding `ensure_packages(['gnupg'])` might be an option? 

But package name differs on e.g. Debian, RedHat and Suse, which the `gnupg` module handles properly.

**Comment by choffee** [comment]
The gnupg module it relies on has not been touched in many years. It might be good to move this into the module.

**Comment by choffee** [comment]
Ah looks like there is now an install_gnupg parmeter in the main branch which fixes it for me. Any chance of a release?

**Comment by ekohl** [comment]
> The gnupg module it relies on has not been touched in many years. It might be good to move this into the module.

Agreed it's poorly maintained and needs attention. Right now it's blocking Puppet 8 support as well.

> Maybe dropping `include gnupg` (it's only installing the package) and adding `ensure_packages(['gnupg'])` might be an option?

It does use more of the module:
https://github.com/voxpupuli/puppet-rvm/blob/246434bafe509d82f85d07e4dd9f980d24e01e32/manifests/system.pp#L39-L47

I don't know if that could be easily replaced by something else.

**Comment by saz** [comment]
I've just suggested to drop the `include gnupg` part, as this isn't adding anything else, except the package.

Maybe https://forge.puppet.com/modules/h0tw1r3/gnupg/readme is a potential replacement?
